### PR TITLE
Scc 2962

### PR DESCRIPTION
**Pull request by charmingduchess**
Removed a recap location and added a location code

### CODE REVIEW

File: vocabularies/csv/locations.csv [hunk 1]
```diff
@@ -1678,6 +1678,7 @@ mao82,Schwarzman Building - Manuscripts & Archives Room 328,SASB - Manuscripts &
 mao92,Schwarzman Building M2 - Manuscripts Division Room 328,SASB M2 - Manuscripts Division - Room 328,Research,,,ma,1107,false,mao,,true,
 maor2,Schwarzman Building - Manuscripts & Archives Room 328,SASB - Manuscripts & Archives Rm 328,Research,,,ma,1107,false,mao,,true,
 map,Schwarzman Building - Map Division Room 117,SASB - Map Division - Rm 117,Research,Research,,ma,1106,false,map,ND,true,map-division
+map08, Schwarzman Building - Map Division Room 177, SASB - Map Division - Rm 117, Research,,,ma,1106,false,map,,true,
```
Comment (nonword) [inline vocabularies/csv/locations.csv:1681]: I'm checking with people, but I think this label may need to include "M1A" qualifier - i.e. "Schwarzman Building - M1A - Map Division Room 117" and "SASB - M1A - Map Division Room 117".

Also, make sure there's no whitespace before the label as it does get rendered into the json-ld. (Which is probably harmless, but could change how a list of locations is sorted, for example.)

**Review by nonword — CHANGES_REQUESTED** [review]
Looking good. A couple things:
 - See note on the new location label
 - We should also update the version declaration in the README
 
I verified your changes using `./vocabularies/scripts/validate-changes` (i.e. `python validate-changes.py recapCustomerCodes` and `python validate-changes.py locations`), which prints out a report on what has changed in the named json-ld file between the current branch and `master`. It shows the csv to json-ld process you ran produced the right changes. (This is a script we've written to address diffing json files. We may want to include it as a step in the README.)

**Comment by charmingduchess** [comment]
Put the changes you requested, only I followed the formatting of the adjacent locations for adding M1A (the other ones didn't have M2 or M1 separated by hyphens).

File: vocabularies/csv/locations.csv [hunk 2]
```diff
@@ -1678,6 +1678,7 @@ mao82,Schwarzman Building - Manuscripts & Archives Room 328,SASB - Manuscripts &
 mao92,Schwarzman Building M2 - Manuscripts Division Room 328,SASB M2 - Manuscripts Division - Room 328,Research,,,ma,1107,false,mao,,true,
 maor2,Schwarzman Building - Manuscripts & Archives Room 328,SASB - Manuscripts & Archives Rm 328,Research,,,ma,1107,false,mao,,true,
 map,Schwarzman Building - Map Division Room 117,SASB - Map Division - Rm 117,Research,Research,,ma,1106,false,map,ND,true,map-division
+map08,Schwarzman Building M1A - Map Division Room 177, SASB M1A - Map Division - Rm 117, Research,,,ma,1106,false,map,,true,
```
Comment (nonword) [inline vocabularies/csv/locations.csv:1681]: Still a leading space in front of the altLabel

**Review by nonword — APPROVED** [review]
Looks good!

**Review by nonword — APPROVED** [review]

**Pull request merged**